plugins/badges-backend/src/types.ts

```diff
@@ -61,3 +61,9 @@ export interface BadgeFactory {
 export interface BadgeFactories {
   [id: string]: BadgeFactory;
 }
+
+export const injectables = {
+  CatalogClient: Symbol.for('CatalogClient'),
+  BadgeBuilder: Symbol.for('BadgeBuilder'),
+  BadgeFactories: Symbol.for('BadgeFactories'),
```

Comment:
Pretty sure we're going to want to namespace these a bit more, stuff like `'plugin.badges-backend.BadgeFactories'`. I'm actually quite confused by the [inversify docs](https://github.com/inversify/InversifyJS/blob/master/wiki/symbols_as_id.md#support-for-symbols) referring to this usage of symbols as a solution to naming collisions compared to using strings. They share an equivalent space, so what gives 😅

You can ofc use `Symbol('...')` instead, but then you end up in dependency duplication hell instead, so I think this is just fine.